pages/Nvidia/_index.md

```diff
@@ -152,107 +146,109 @@ env = ELECTRON_OZONE_PLATFORM_HINT,auto
 This has been confirmed to work on Vesktop, VSCodium, Obsidian and will probably
 work on other Electron apps as well.
 
-For other apps, including CEF apps, you will need to launch them with these flags:
+For other apps, including CEF apps, you will need to launch them with these
+flags:
 
 ```sh
 --enable-features=UseOzonePlatform --ozone-platform=wayland
 ```
 
-To do this easily for Spotify, Arch Linux has a `spotify-launcher` packages
-in their official repos. You should use that instead of the `spotify`
-package in the AUR. Then, enable the Wayland backend in
-`/etc/spotify-launcher.conf` by uncommenting this line:
+To do this easily for Spotify, Arch Linux has a `spotify-launcher` package in
+their official repos. You should use that instead of the `spotify` package in
+the AUR. Then, enable the Wayland backend in `/etc/spotify-launcher.conf` by
+uncommenting this line:
```

Comment:
```suggestion
To do this easily for Spotify, Arch Linux has a `spotify-launcher` package in
their official repos. You should use that instead of the `spotify` package in
the AUR. Then, copy the `/etc/spotify-launcher.conf` file to `$HOME/.config/
and enable the Wayland backend in that file by uncommenting this line:
```